pylint/checkers/variables.py

```diff
@@ -1988,15 +1983,15 @@ def _is_first_level_self_reference(
         refers to its own class.
 
         Return values correspond to:
-            0 = Continue
+            0 = Continue and consume names
```

Comment:
Maybe we could rename the enum attribute and document this in the enum directly.